### ISSUE TITLE
All wallets and coins disappeared

### ISSUE THREAD

**Issue by krstofu** [bitpay/copay#6285]
I'm using Copay on Windows Phone 10. Today when I opened the app all wallets and coins had disappeared and I was instead greeted with the welcome screen where you can either choose to "Get Started" or restore a single wallet from backup.

What happened to all my wallets? I have automatic backups set for this app but it appears all my data has been wiped somehow. Perhaps the developers have pushed an update as I have noticed the UI is slightly different than a few days ago when it was working?

How can the data and the wallets be restored without having to restore each wallet separately? 

Why did the app wipe all my wallets?

**Comment by gabrielbazan7** [comment]
Sorry, not sure what could happen. Do have the words of your wallets ? What do you mean by automatic backups ?

**Comment by krstofu** [comment]
I have the backup words for some of the wallets but not all. By automatic backups I mean Windows 10 Phone has an option in the backups settings where you can "Choose which apps to back up". It says Copay Bitcoin Wallet back up is on.

**Comment by JDonadio** [comment]
Not sure if that functionality really backed your wallets up. Be sure to make your backups manually 😉

**Comment by krstofu** [comment]
So you pushed a big release that wipes everything from the app, all your wallets and coins?????? OMG. Yes, it's my own fault for not backing up all the wallets, but that would be the one thing you would test before pushing a big update. THE ONE THING.

**Comment by matiu** [comment]
Hello, 

I am sorry you are having problems with Copay. Installing a new version of the app should keep the old wallets, would it help if we provide the old app file so you can try restore the old wallet by installing it manually? Did you have significant amount of money on the wallet with no backup?

matías

**Comment by krstofu** [comment]
By "installing a new version" do you mean uninstalling then reinstalling the app? Yes there was significant amount of money on the wallets with no backup so any help recovering it would be highly appreciated, thanks.

**Comment by matiu** [comment]
No, by installing a new version I mean just upgrading the app. Uninstalling will for sure delete the wallets.

We will work to get you the old version's windows app file, so you can try to install it manually. We will update this ticket when we have it.

**Comment by krstofu** [comment]
Yes I thought so, I was just checking. But as far as I can see I'm already running the latest version. Automatic updates are on. It says the app updated yesterday, which is probably when the issue occurred. The upgrade broke it.

**Comment by mm2k2us** [comment]
I have exactly the same problem. I started Copay to check my wallet but the update has wiped everything which is not cool because I am not at home to restore my wallet with the passphrase. Any chance the old app file would work for me too?

**Comment by matiu** [comment]
@mm2k2us do you have the backup words? (12 words)

**Comment by mm2k2us** [comment]
I should have them at home. But I am on vacation and cannot access my wallet now.

**Comment by xpetrv** [comment]
same issue here, everything disappeared within last week.... what should I do?

**Comment by JDonadio** [comment]
@xpetrv in your case just import your wallet again using your 12 words.

**Comment by rbolyu** [comment]
I've also lost a significant amount of funds due to the upgrade that broke backwards compatibility.  Why would you release an update which wasn't compatible with previous versions?  I cannot find my twelve backup words for this wallet.  I assumed having the wallet.dat files backed up was enough.  Bitpay please release the former version of the Copay app and instructions so those of us stuck with funds in previously operational wallets can access those funds!

**Comment by mm2k2us** [comment]
Just wanted to kindly check if there is any progress on the old app file? 

Even though I might be able to recover the amounts using the passphrase, I am still disappointed that a new auto-update release completely wiped all existing information.

**Comment by Schoepi** [comment]
Das selbe bei mir. Als ich die Copay App nach ein paar  Tagen wieder öffnen wollte, war "ungefragt" ein update durchgeführt worden. Beim Starten wurden dann gutgemeinte Warnungen gelistet. Aber kein Wort darüber verloren, dass die Wallets mit samt den Adressen weg sind und nicht übernommen werden.
Dieses Vorgehen ist eine Zumutung für alle Benutzer. Nicht nur dass nun einige ihre Bitcoins definitiv verloren haben. Auch diejenigen welche die Passphrase gesichert haben, werden die Passphrasen hoffentlich, kaum mit auf die Reise mit nehmen und stehen beim shoppen od. einem Restaurantbesuch sau blöd vor der Kasse.
Ich bin äusserst verärgert über dieses vorgehen.

**Comment by Derrick-** [comment]
Has this issue been resolved in the latest version?

**Comment by krstofu** [comment]
The issue has not been resolved. The developers push updates that delete the entire contents of your wallet. You lose everything. The incompetence is mind-boggling. Initially they said they would provide the old app file but they never did. Do not use Copay, it's not safe.

**Comment by platedodev** [comment]
I'm using Copay v3.4.0 and I'm trying to use my 12 words on Electrum but it show a 0 BTC balance which is happening because I'm getting differents address than I'm getting on my Copay wallet. Are there any bug on derivation?.

**Comment by munirh** [comment]
I have Visa BitPay card. They attach a CoPay wallet with your account. They try and make it easy to access your funds....try too...I transferred 2k worth of bitcoin from my Coinbase account to CoPay Wallet. The trasnfer went through just fine. When I tried to move the bitcoin over from CoPay wallet to my Visa Card attached to it, it kept giving me an error. I deleted the app and reinstalled the app (I dont know why I did this). I am running into the same issue as everyone else. I do not have the 12 words. 

Is there anything I can do?!

**Comment by kaaewhy** [comment]
I lost access to my bitcoin on my copay wallet as well for my windows phone. How can I get the copay version previous to june 2017?
